daq/runner.py

```diff
@@ -578,8 +578,6 @@ def _activate_device_group(self, device):
         LOGGER.info('Gateway for device group %s not found, initializing base %d...',
                     device.group, set_num)
         if device.dhcp_mode == DHCPMode.EXTERNAL:
-            # TODO to be removed.
```

Comment:
roughly indicate why

Reply:
The TODO should indicate why -- also the comment you added doesn't explain "why" -- it's saying what that line is doing.  Why should this be removed eventually -- i.e. under what condition should it be removed?